build.sh

```diff
@@ -74,9 +80,13 @@ function printHelp() {
     {
         printf " \t%s\t%s\n" "--test" "to run the tests"
         printf " \t%s\t%s\n" "--testUi" "to run the tests including UI tests"
+        printf " \t%s\t%s\n" "--installCore" "install JMC core"
```

Comment:
If trying to keep consistency, the `--installCore` is missing the "to" in it's description; it has it in the .bat file though

Reply:
corrected